paasta_tools/cli/cmds/logs.py

```diff
@@ -462,15 +461,16 @@ def prettify_log_line(
             )
         else:
             return (
-                "%(timestamp)s %(component)s %(cluster)s %(instance)s - %(level)s%(message)s"
+                "%(timestamp)s %(component)s %(cluster)s %(instance)s - %(pod)s%(container)s"
                 % (
                     {
                         "timestamp": prettify_timestamp(parsed_line["timestamp"]),
                         "component": prettify_component(parsed_line["component"]),
                         "cluster": "[%s]" % parsed_line["cluster"],
                         "instance": "[%s]" % parsed_line["instance"],
-                        "level": "%s" % pretty_level,
-                        "message": parsed_line["message"],
+                        "pod": "%s" % parsed_line["pod_name"],
+                        "container": "%s"
+                        % re.findall(r"\(.*?\)", parsed_line["message"])[0],
```

Comment:
This looks to me like it is replacing the `level` and `message` with `pod` and `container`.  Also for app_output log lines I think you have access to the container name in the `container_name` field, so you don't really need any regex shenanigans.

Most importantly, though, it looks to me like the hostname is included in the `message` field (based on the one service whose log lines I peeked at), meaning that we can't really get rid of it...so I am wondering if this ticket actually makes sense.

Reply:
I guess it probably still makes sense to get rid of level, though?

Reply:
+1

Reply:
Only removing level and keeping message as long as pod and container.

Follow up actions: Enforce user to input only one of each (service, instance and cluster). By doing so, we will be able to remove those fields from the log lines, and thus saving space for both pod and container.